pkg/lib/serverTime.js

```diff
@@ -439,63 +606,142 @@
     );
 }
 
-function ChangeSystimeBody({ state, errors, change }) {
-    const [zonesOpen, setZonesOpen] = useState(false);
-    const [modeOpen, setModeOpen] = useState(false);
-
-    const {
-        time_zone, time_zones,
-        mode,
-        manual_date, manual_time,
-        ntp_supported, custom_ntp
-    } = state;
-
+const DynamicServerList = ({ servers, errors, change }) => {
     function add_server(event, index) {
-        custom_ntp.servers.splice(index + 1, 0, "");
-        change("custom_ntp", custom_ntp);
+        servers.splice(index + 1, 0, "");
+        change();
         event.stopPropagation();
         event.preventDefault();
         return false;
     }
 
     function remove_server(event, index) {
-        custom_ntp.servers.splice(index, 1);
-        change("custom_ntp", custom_ntp);
+        servers.splice(index, 1);
+        change();
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17992-20230120-130737-0dda615f-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)